### ISSUE TITLE
Importing same entities from different locations

### ISSUE THREAD

**Issue by shivangg** [webhintio/hint#2315]
In `hint-apple-touch-icons`, imports are like:

```
import { HintContext, IHint, NetworkData } from 'hint'  
```

whereas in `create-hint` template makes them get imported like: 

```
import { HintContext } from 'hint/dist/src/lib/hint-context';
import { IHint, NetworkData } from 'hint/dist/src/lib/types';
```

Looking over other packages,
`hint-manifest-exists`, `no-p3p` also takes the 1st approach.
`hint-axe`, `hint-manifest-is-valid` takes the 2nd approach.

Is one better than the other?
What's the reason for this non-uniformity?

**Comment by molant** [comment]
The first one is the preferred option. We are refactoring the utils at the moment and haven't changed that yet in the create packages. In the PRS section there's a draft one to flatten even more that from @antross.

Hopefully we'll get to it next sprint 😔

**Comment by shivangg** [comment]
Okay. Also, fixing that in the `create-hint` package will prevent new hint package from having the imports like the 2nd approach.

**Comment by molant** [comment]
Yes, we intend on fixing that once the [flatten PR is merged](https://github.com/webhintio/hint/pull/2216) but if you want to go ahead and fix it in this one the change will be more than welcomed 😊

**Comment by antross** [comment]
Also worth noting is that the second form is required for any hints we want to include as part of `extension-browser` until we can do tree-shaking during the bundling process (otherwise we end up pulling in much more than we need). That said, I'd still prefer to have the create-hint package starting with the first approach.
